### ISSUE TITLE
Sending completion user feedback

### ISSUE THREAD

**Issue by jprester** [EleutherAI/text-generation-testing-ui#14]
can we add like a "did you like this completion" prompt and log that too?

**Comment by kingoflolz** [comment]
I think this is a great idea, we can likely defer this to the next model launch?

**Comment by jprester** [comment]
Agree! I will add this as a ticket to the new project plan: https://github.com/EleutherAI/text-generation-testing-ui/projects/1
